bin/var-dump-server

```diff
@@ -0,0 +1 @@
+../vendor/symfony/var-dumper/Resources/bin/var-dump-server
```

Comment:
Why include this.

Reply:
It's an update from Symfony's var-dumper component